src/components/v2/shared/SplitItem.tsx

```diff
@@ -62,24 +67,33 @@ export default function SplitItem({
   const itemFontSize = isMobile ? '0.9rem' : 'unset'
 
   const JuiceboxProjectBeneficiary = () => {
+    const getProjectTooltip = () => {
+      return split.projectId
+        ? t`Juicebox V2 project with ID ${parseInt(split.projectId)}`
+        : t`Juicebox V2 project`
+    }
+    const getProjectLabel = () => {
+      if (handle) {
+        return t`@${handle}`
+      }
+      if (split.projectId) {
+        return t`Project ${parseInt(split.projectId)}`
+      }
+      return t`Unknown Project`
+    }
     return (
       <div>
-        {/* TODO figure out project "handles" with ENS resolution */}
-
         <div style={{ fontWeight: 500 }}>
-          <Tooltip
-            title={<Trans>Juicebox V2 project with ID {split.projectId}</Trans>}
+          <Link
+            href={v2ProjectRoute({ projectId: split.projectId })}
+            target="_blank"
           >
-            <Link
-              href={v2ProjectRoute({ projectId: split.projectId })}
-              target="_blank"
-            >
+            <Tooltip title={getProjectTooltip()}>
```

Comment:
**question** i seem to recall `Link` needing `<a>` as its direct child. Any particular reason to move the tooltip within?

Reply:
Wrapped it in a `span` and made `a` direct child of `link`. Need this so that `a` gets the `href` attribute. Then it behaves like a real link (notice bottom-left of screenshot)


<img width="640" alt="Screen Shot 2022-07-28 at 12 39 36 PM" src="https://user-images.githubusercontent.com/12551741/181412023-167f07d5-f456-4fb0-ab16-b8d566a72d23.png">